src/services/axiosInstance.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const API = axios.create({
+  baseURL: "https://cada098223b17ac7900d.free.beeceptor.com/todo/",
```

Comment:
The base URL refers to the root part of the URL. We need to append the specific route at the point where the API is being called.

```suggestion
  baseURL: "https://cada098223b17ac7900d.free.beeceptor.com/",
```

Reply:
The reason I included /todo/ in the baseURL is because in Beeceptor I created the mock endpoint at /todo. So the full URL is https://cada098223b17ac7900d.free.beeceptor.com/todo/.

To match that, the baseURL needs to include /todo/ so that calls like API.get('/') correctly hit the mock endpoint.